crates/astria-conductor/src/executor/mod.rs

```diff
@@ -265,8 +207,10 @@ impl<TBlockChannel, TRollupAddress, TRollupId, TShutdown>
 }
 
 pub(crate) struct Executor {
-    /// Channel on which executor commands are received.
-    block_channel: Receiver,
+    celestia_rx: mpsc::UnboundedReceiver<Vec<SequencerBlockSubset>>,
+    celestia_tx: mpsc::WeakUnboundedSender<Vec<SequencerBlockSubset>>,
```

Comment:
why `WeakUnboundedSender` vs `UnboundedSender`?

Reply:
is it because `Executor` will never send on these tx channels? if so can you add a comment

Reply:
After rereading https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.WeakUnboundedSender.html I realized that I misunderstood `WeakSender`. I thought upgrading fails if there are no other senders (either weak or full/upgraded), but that actually does not make sense. Upgrading fails if there are no other full senders, so what I implemented here will never work.

Thank you for flagging, I will fix this!

Reply:
I have fixed this in PR https://github.com/astriaorg/astria/pull/711